### PR TITLE
breaking changes labeler workflow

### PR DESCRIPTION
**Pull request by acn-sbuad**
## Description
Labeler workflow failing since upgrade to v5 of action

### CODE REVIEW

File: .github/workflows/pr-labeler.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - uses: actions/checkout@v4
```
Comment (SandGrainOne) [inline .github/workflows/pr-labeler.yml:17]: This is adding checkout, but not changing labeler.

Comment (acn-sbuad) [reply .github/workflows/pr-labeler.yml:17]: only the config requires changing. v5 of labeler includes the breaking changes, but config was not updates when action was patched

Comment (SandGrainOne) [reply .github/workflows/pr-labeler.yml:17]: Okay, so the checkout step doesn't require any input?

Comment (acn-sbuad) [reply .github/workflows/pr-labeler.yml:17]: no

**Review by SandGrainOne — APPROVED** [review]

**Pull request merged**